tests/console/yast2_lan.pm

```diff
@@ -75,9 +75,7 @@ sub run() {
 
     wait_still_screen;
     clear_console;
-    script_run('echo $?');
-    script_run('hostname');
-    assert_screen 'test-yast2_lan-2';
+    script_run "hostname|grep $hostname";
```

Comment:
why not to use assert_script_run? otherwise how do you guarantee it has the right hostname there :)